mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -655,6 +655,15 @@ def delete_run(self, run_id):
             run.deleted_time = get_current_time_millis()
             session.add(run)
 
+    def delete_runs(self, run_ids):
+        with self.ManagedSessionMaker() as session:
+            conditions = or_(*[SqlRun.run_uuid == run_id for run_id in run_ids])
```

Comment:
Can we use `IN` clause ? otherwise the sql predicates are too many and might cause issues.